src/app/components/search-additional-buttons/search-additional-buttons-nav.component.html

```diff
@@ -1,6 +1,6 @@
 <ul class="nav search-additional-buttons__nav">
     <ng-template ngFor let-searchAdditionalData [ngForOf]="searchAdditionalButtonsData">
-        <li class="nav-item mr-2 pt-2">
+        <li class="nav-item mr-2 pt-2" [appFeatureFlag]="searchAdditionalData.id">
```

Comment:
I don't think we have declared this feature

Reply:
If we need it, make it a subfeature of `search`

Reply:
Moved to a subfeature of search